src/data/tags.ts

```diff
@@ -37,6 +37,10 @@ export const tagData: TagInterface = {
     label: 'Medical',
     color: 'red.600',
   },
+  furniture: {
+    label: 'Furniture',
+    color: 'lime.900',
```

Comment:
There is no `lime.900` color.
https://chakra-ui.com/docs/styled-system/theme#colors